src/style.css

```
@@ -0,0 +1,83 @@
+body {
+  margin: 0;
+  font-family: Arial, sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.box {
+  height: 300px;
+  min-width: 300px;
+  display: flex;
+  flex: 1 1 33.33%;
+  align-items: center;
+  justify-content: center;
+  color: #fff;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+}
+
+.box-1 {
+  background-color: red;
+  order: 1;
+}
+
+.box-2 {
+  background-color: #c00;
+  order: 2;
+}
+
+.box-3 {
+  background-color: #900;
+  order: 3;
+}
+
+.box-4 {
+  background-color: #600;
+  order: 4;
+}
+
+.box-5 {
+  background-color: #300;
+  order: 5;
+}
+
+.box-6 {
+  background-color: #000;
+  order: 6;
+}
+
+@media (min-width: 600px) {
+  .container {
+    flex: 1 1 100%;
```

зачем переназначать то что уже назначено?
```suggestion
    flex-basis: 100%;
```